### DISCUSSION TITLE
filtering: OR query across fields

### DISCUSSION THREAD

**Discussion by casperdcl** [community/community#41017] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-items currently allows `field:one,two` (i.e. `field:one` OR `field:two`). It does not allow OR queries across different fields (`fieldA:one OR fieldB:two`).

See also [SO#73406538](https://stackoverflow.com/q/73406538/3896283).

**Comment by stilliard** [comment]
This would be super useful, for seeing where you're assigned to an issue or where you're the reviewer of a pull request in the same view. 
Plenty other really useful use cases too.

**Reply by mhsnook** [reply]
exactly! came here to say this! I can use `michaelsnook` (without the `assignee:` field part) but that will not with with aliases like `@me` so there's no way to create a board: `cycle:@current (assignee:@me OR reviewer:@me)` which is, like, an extremely good use case for the projects product :v:

**Reply by andrewawilley** [reply]
same need, my usecase is (assignee:@me OR no:assignee)
